src/DssSpell.sol

```diff
@@ -42,22 +67,96 @@ contract SpellAction {
     //
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
-    uint256 constant        ZERO_PCT_RATE = 1000000000000000000000000000;
-    uint256 constant         ONE_PCT_RATE = 1000000000315522921573372069;
-    uint256 constant         TWO_PCT_RATE = 1000000000627937192491029810;
     uint256 constant         SIX_PCT_RATE = 1000000001847694957439350562;
-    uint256 constant         TEN_PCT_RATE = 1000000003022265980097387650;
-    uint256 constant FOURTYEIGHT_PCT_RATE = 1000000012431573129530493155;
 
     function execute() external {
 
-        // Set the ETH-A stability fee
-        //
-        // Previous: 2%
-        // New: 0%
-        //JugAbstract(MCD_JUG).drip("ETH-A"); // drip right before
-        //JugAbstract(MCD_JUG).file("ETH-A", "duty", ZERO_PCT_RATE);
-
+        /*** ETH-B Collateral Onboarding ***/
+
+        //   $ seth --to-bytes32 $(seth --from-ascii "ETH-B")
+        //   0x4554482d42000000000000000000000000000000000000000000000000000000
+        bytes32 ilk = "ETH-B";
+
+        // Sanity checks
+        require(GemJoinAbstract(MCD_JOIN_ETH_B).vat() == MCD_VAT, "join-vat-not-match");
+        require(GemJoinAbstract(MCD_JOIN_ETH_B).ilk() == ilk, "join-ilk-not-match");
+        require(GemJoinAbstract(MCD_JOIN_ETH_B).gem() == ETH, "join-gem-not-match");
+        require(GemJoinAbstract(MCD_JOIN_ETH_B).dec() == 18, "join-dec-not-match");
+        require(FlipAbstract(MCD_FLIP_ETH_B).vat() == MCD_VAT, "flip-vat-not-match");
+        require(FlipAbstract(MCD_FLIP_ETH_B).cat() == MCD_CAT, "flip-cat-not-match");
+        require(FlipAbstract(MCD_FLIP_ETH_B).ilk() == ilk, "flip-ilk-not-match");
+
+        // Add the new flip and join to the Chainlog
+        ChainlogAbstract(CHAINLOG).setAddress("MCD_JOIN_ETH_B", MCD_JOIN_ETH_B);
+        ChainlogAbstract(CHAINLOG).setAddress("MCD_FLIP_ETH_B", MCD_FLIP_ETH_B);
+        ChainlogAbstract(CHAINLOG).setVersion("1.1.3");
+
+        // Set the TOKEN PIP in the Spotter
+        SpotAbstract(MCD_SPOT).file(ilk, "pip", PIP_ETH);
+
+        // Set the TOKEN-LETTER Flipper in the Cat
```

Comment:
Personally I'd do `s/TOKEN-LETTER/ETH-B` to make the comments clearer, but that's just me.